packages/calypso-color-schemes/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release Notes
 
+## 2.2.0
+
+* Introduce Google Color
```

Comment:
Past release notes specifically mention added or removed properties. Suggested change:

```
## 2.1.2

* Added Google brand color:
    * `--color-google`
```

Reply:
Thank you for the feedback. Done in 5483ec93a467ee8f3675821386635dfe25af79ca